modules/nextflow/src/main/groovy/nextflow/executor/FluxExecutor.groovy

```diff
@@ -87,7 +89,8 @@ class FluxExecutor extends AbstractGridExecutor {
 
         // the requested partition (a.k.a queue) name
         if( task.config.queue ) {
-            result << '--queue=' + (task.config.queue.toString())
+            log.debug "Queue parameter not supported by standalone flux."
```

Comment:
This is also still available:

```
 -q, --queue=NAME            Submit a job to a specific named queue
```

Reply:
Reply from the original author:

---

Uhm ... `--queue` doesn't fit here anyway because there is no queue (names) when you run "flux start" manually. So, I didn't know what to do with it.

(To explain: As I tried to outline previously once ... I start a flux instance inside a Slurm job ... as user. For jobfarming. So there is no centrally managed flux instance with different queues etc. Just flux-in-slurm ... )

---

I'm not quite sure what's going on here. But either way, I think that they can just omit `queue` in the Nextflow config and this won't be used. So no need to delete from the executor source 👍🏻